### PR TITLE
NEXUS-29515 - At least skip trying to deploy a snapshot while this plugin cannot support it

### PR DESCRIPTION
**Pull request by stephenc**

### CODE REVIEW

**Comment by thejosh00** [comment]
This looks useful. +1 from me

File: maven-plugin/src/main/java/org/sonatype/nexus/maven/staging/PrepareDeployMojo.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package org.sonatype.nexus.maven.staging;
```
Comment (ybulatnikov) [inline maven-plugin/src/main/java/org/sonatype/nexus/maven/staging/PrepareDeployMojo.java:1]: License header is missing. Please run this command to add the header: `./headers.sh format`

**Review by ybulatnikov — CHANGES_REQUESTED** [review]

**Comment by bobotimi** [comment]
This PR has been open for six months, is it still needed?

**Comment by stephenc** [comment]
Yes still required for us at least. Impossible to have deploy work for both
-SNAPSHOT and releases from the regular command line of `mvn deploy`
without this or an equivalent solutuion

**Comment by tsawan** [comment]
@stephenc can you sign the CLA so we can proceed to merge this PR? https://sonatypecla.herokuapp.com/sign-cla

**Comment by stephenc** [comment]
image

It won't let me sign it again

**Comment by stephenc** [comment]
As it it's been signed for over a year now, so I don't see what the block is

**Comment by stephenc** [comment]
@brianf how long does it take you guys to merge a PR! Feel free to rework this PR as you see fit.

**Comment by tsawan** [comment]
merged and released 1.0.6

**Pull request closed without merging**